src/styles/watch.scss

```diff
@@ -0,0 +1,43 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $minuteHandHeight * 2 + $emtywatchSpace;
+  height: $minuteHandHeight * 2 + $emtywatchSpace;
+  border: 10px solid black;
+  border-radius: ($minuteHandHeight * 2 + $emtywatchSpace) / 2;
+  background: linear-gradient(to right, #ff4e50, #f9d423);
+
+  &__minute-hand {
+    position: relative;
```

Comment:
for these two hands you can add properties such as position, width, border-radius, transform-origin into a mixin like "hands-basic-configuration".

Reply:
good point, use it